e2e/tests/upgrades/genesis_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 
+	interchainaccounts "github.com/cosmos/ibc-go/e2e/tests/interchain_accounts"
```

Comment:
this doesn't cause a circular dependency right?

Reply:
yeah, I've checked it. It doesn't cause a circular dependency